docs/src/standalone/apps/leaf_an.jl

```diff
@@ -38,10 +38,10 @@ function ParamViz.parameterisation(
     Γstar = co2_compensation(Γstar25, ΔHΓstar, T, To, R)
     medlynt = 1 + g1 / sqrt(VPD)
     ci = intercellular_co2(ca, Γstar, medlynt)
-    Aj = light_assimilation(Canopy.C3(), J, ci, Γstar)
+    Aj = light_assimilation(1.0, J, ci, Γstar)
```

Comment:
for clarity here, perhaps we could define is_c3 = 1.0 and then use that in the functions